app/assets/javascripts/shared/dradis_datatable.js

```diff
@@ -2,30 +2,68 @@ class DradisDatatable {
   constructor(tableElement) {
     this.$table = $(tableElement);
     this.dataTable = null;
+    this.tableHeaders = Array.from(this.$table[0].querySelectorAll('thead th, thead td'));
     this.init();
   }
 
   init() {
-    var dataTable = this.$table.DataTable({
+    // Remove dropdown option for <th> columns that has data-colvis="false" in colvis button
```

Comment:
comment explains what the code do (the code does that already) can we explain the why instead?

Reply:
It removes the option from the dropdown.

![image](https://user-images.githubusercontent.com/18374402/118445084-8ef21b80-b720-11eb-9134-26d27e371e8c.png)